flytekit/extras/sqlalchemy/task.py

```diff
@@ -54,13 +57,12 @@ def __init__(
     ):
         output_schema = output_schema_type if output_schema_type else FlyteSchema
         outputs = kwtypes(results=output_schema)
-        self._uri = task_config.uri
-        self._connect_args = task_config.connect_args or {}
-        self._secret_connect_args = task_config.secret_connect_args
 
         super().__init__(
             name=name,
             task_config=task_config,
+            container_image="public.ecr.aws/nuclyde/sqlalchemy:ce755dd8886cd66af1821051682a5b0181bc3626",
```

Comment:
Let's use ghcr. Looks much better I think.

Reply:
Can you give me access to push an image to ghcr? @kumare3